include/ginkgo/core/preconditioner/isai.hpp

```diff
@@ -57,22 +57,30 @@ namespace preconditioner {
 /**
  * This enum lists the types of the ISAI preconditioner.
  *
- * ISAI can either generate a lower triangular matrix, or an upper triangular
- * matrix.
+ * ISAI can either be generated for a general square matrix, a lower triangular
+ * matrix, an upper triangular matrix or a spd matrix.
```

Comment:
from the pronounciation, I would assume this is similar to `an essential`
```suggestion
 * matrix, an upper triangular matrix or an spd matrix.
```